internal/plugins/helm/v1/plugin.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/operator-framework/operator-sdk/internal/plugins"
 )
 
-const pluginName = "helm" + plugins.DefaultNameQualifier
+const pluginName = "base.helm" + plugins.DefaultNameQualifier
```

Comment:
we cannot change the plugins name. The layout needs still showing the same name.

Reply:
The `"helm.sdk.operatorframework.io/v1"` plugin bundle hides this. `"base.helm.*"` is not used anywhere (neither in the layout nor in the flags).

Reply:
NIT: We need to comment on that. Provide an explanation in the code only.